File: src/main/java/de/taschi/bulkgpsviewer/settings/dto/SettingsColor.java
package de.taschi.bulkgpsviewer.settings.dto;

public class SettingsColor {
	
	public SettingsColor() {
		super();
		r = 0;
		g = 0;
		b = 0;
	}
	
	public SettingsColor(int r, int g, int b) {
		super();
		this.r = r;
		this.g = g;
		this.b = b;
	}

	private int r, g, b;

	public int getR() {
		return r;
	}

	public void setR(int r) {
		this.r = r;
	}

	public int getG() {
		return g;
	}

	public void setG(int g) {
		this.g = g;
	}

	public int getB() {
		return b;
	}

	public void setB(int b) {
		this.b = b;
	}
	
	
}

File: src/main/java/de/taschi/bulkgpxviewer/settings/dto/SettingsColor.kt
package de.taschi.bulkgpxviewer.settings.dto

/*-
 * #%L
 * bulkgpxviewer
 * %%
 * Copyright (C) 2021 S. Hillebrand
 * %%
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as
 * published by the Free Software Foundation, either version 3 of the
 * License, or (at your option) any later version.
 * 
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 * 
 * You should have received a copy of the GNU General Public
 * License along with this program.  If not, see
 * <http://www.gnu.org/licenses/gpl-3.0.html>.
 * #L%
 */
class SettingsColor {
    constructor() : super() {
        r = 0
        g = 0
        b = 0
    }

    constructor(r: Int, g: Int, b: Int) : super() {
        this.r = r
        this.g = g
        this.b = b
    }

    var r: Int
    var g: Int
    var b: Int
}